crates/ruff_linter/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -678,7 +678,35 @@ fn check_duplicates(checker: &mut Checker, values: &Expr) {
     }
 }
 
-fn handle_single_name(checker: &mut Checker, expr: &Expr, value: &Expr) {
+/// Generate [`Edit`]s to unpack single-element lists or tuples in the given [`Expr`].
+/// For instance, `[(1,) (2,)]` will be transformed into `[1, 2]`.
+fn unpack_single_element_items(checker: &Checker, expr: &Expr) -> Vec<Edit> {
+    let (Expr::List(ast::ExprList { elts, .. }) | Expr::Tuple(ast::ExprTuple { elts, .. })) = expr
+    else {
+        return vec![];
+    };
+
+    let mut edits = Vec::with_capacity(elts.len());
+    for value in elts {
```

Comment:
What does it mean if `elts` contain more than one element? Is it possible? Should we error or not provide a fix in that case?

Reply:
That's possible. That would make `pytest.mark.parametrize` throw. A fix should not be provided.

Reply:
Would you mind adding a a test for this and handle it appropriately (by bailing to create a fix)?

Reply:
Added!